index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio</title>
+</head>
+<body>
+    <div class="interface">
+        <header class="header">
+            <img class="logo" src="FireBorne Entertainment.png" alt="My Logo">
+            <img class="hamburger" src="./Union.png" alt="Hamburger">
+        </header>
+        </br>
+        <h1>I'm FirePicaso. Glad to see You!</h1>
+        <p>
+            I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <h2>Lets connect</h2>
```

Comment:
Well done here as well by using hx tags for titles and p for paragraph 👍🏼  , and it's a good practice to wrap these elements in a section as they represent a separate section in the page this will make your code clean and readable when you add other blocks